### PR TITLE
Show message with suggestion to try legacy FE in case of conversion error

### PR DESCRIPTION
**Pull request by popovaan**
Description: 
Added showing of message with suggestion to try legacy FE in case of conversion with Tensorflow FE failed.
Moved checking of exception types to common part in _convert() method.

Ticket: 105537

Code:
* [x]  Comments
* [x]  Code style (PEP8)
* [x]  Transformation generates reshape-able IR - N/A
* [x]  Transformation preserves original framework node names - N/A
* [x]  Transformation preserves tensor names - N/A

Validation:
* [x]  Unit tests
* [x]  Framework operation tests - N/A
* [x]  Transformation tests - N/A
* [x]  Model Optimizer IR Reader check - N/A

Documentation:
* [x]  Supported frameworks operations list - N/A
* [x]  Guide on how to convert the **public** model - N/A
* [x]  User guide update - N/A

### CODE REVIEW

File: tools/mo/openvino/tools/mo/utils/telemetry_utils.py [hunk 1]
```diff
@@ -103,3 +103,11 @@ def get_tid():
     This function returns the ID of the database to send telemetry.
     """
     return telemetry_params['TID']
+
+
```
Comment (rkazants) [inline tools/mo/openvino/tools/mo/utils/telemetry_utils.py:107]: It is better to simulate a crash of TF FE by registering failing translator for some simple operation.

**Review by rkazants — APPROVED** [review]

File: tools/mo/openvino/tools/mo/utils/telemetry_utils.py (hunk 1 above)
Comment (rkazants) [reply tools/mo/openvino/tools/mo/utils/telemetry_utils.py:107]: This is up to you.

File: tools/mo/openvino/tools/mo/utils/telemetry_utils.py [hunk 2]
```diff
@@ -103,3 +103,11 @@ def get_tid():
...
+def send_conversion_result(conversion_result: str, need_shutdown=True):
+    t = tm.Telemetry()
+    t.send_event('mo', 'conversion_result', conversion_result)
+    t.end_session('mo')
+    if need_shutdown:
+        t.force_shutdown(1.0)
```
Comment (rkazants) [inline tools/mo/openvino/tools/mo/utils/telemetry_utils.py:113]: I think this is not needed

File: tools/mo/unit_tests/moc_tf_fe/check_info_messages_test.py [hunk 3]
```diff
@@ -68,6 +70,21 @@ def test_api20_only(self, mock_argparse):
         tf_fe_message_found = get_tf_fe_message() in std_out
         assert tf_fe_message_found
 
+    @patch('openvino.tools.mo.convert_impl.driver', side_effect=Exception('MESSAGE'))
+    def run_fail_tf_fe(self, mock_driver):
+        from openvino.tools.mo import convert_model
+        convert_model(create_tf_model(), silent=False)
```
Comment (rkazants) [inline tools/mo/unit_tests/moc_tf_fe/check_info_messages_test.py:76]: please use input_model = os.path.join(path, "test_models", "model_int32.pbtxt")

File: tools/mo/openvino/tools/mo/utils/telemetry_utils.py (hunk 2 above)
Comment (popovaan) [reply tools/mo/openvino/tools/mo/utils/telemetry_utils.py:113]: send_conversion_result() is used twice in 944 and 975 in convert_impl.py

**Pull request merged**